python/lsst/pipe/tasks/postprocess.py

```diff
@@ -204,7 +204,25 @@ def writeMetadata(self, dataRefList):
         pass
 
 
-class WriteSourceTableConfig(pexConfig.Config):
+class WriteSourceTableConnections(pipeBase.PipelineTaskConnections,
+                                  dimensions=("instrument", "visit", "detector")):
+
+    catalog = pipeBase.connectionTypes.Input(
+        doc="Input full-depth catalog of sources produced by CalibrateTask",
+        name="src",
+        storageClass="SourceCatalog",
+        dimensions=("instrument", "visit", "detector")
+    )
+    outputCatalog = pipeBase.connectionTypes.Output(
+        doc="Catalog of sources, `src` in Parquet format",
+        name="source",
+        storageClass="DataFrame",
+        dimensions=("instrument", "visit", "detector")
+    )
+
```

Comment:
So for DM-27164 (and this will need to be rebased of course), I defined `from lsst.pipe.base import connectionTypes` so you can use just `connectionTypes` here.

Reply:
It's not obvious to me that `connectionTypes` is more readable than `pipeBase.connectionTypes` when there are other `pipeBase` objects abounding, but sure, I'll change them all.

Reply:
I don't want to makework!  But we don't have any standard convention on this, and that's unpleasant.